robottelo/hosts.py

```diff
@@ -457,7 +486,8 @@ def configure_puppet(self, rhel_repo=None, proxy_hostname=None):
         # sat6 under the capsule --> certifcates or on capsule via cli "puppetserver
         # ca list", so that we sign it.
         self.execute('puppet agent -t')
-        ssh.command(cmd='puppetserver ca sign --all', hostname=proxy_hostname)
+        with Host(proxy_hostname) as proxy_host:
```

Comment:
where is this context manager coming from?

Reply:
My imagination, Host doesn't have its own context manager, just VMBroker.